build1.sh

```diff
@@ -3,14 +3,41 @@
 set -e
 set -x
 
+WITH_STACKTRACE=yes
+
+if [[ -n "${CC}" ]]; then
+  CC=$(which "${CC}")
+fi
+
+if [[ -n "${CXX}" ]]; then
+  CXX=$(which "${CXX}")
+fi
+
+if [[ -n "${CONDA_PREFIX}" ]] && [[ "$(echo ${CC} | grep -m 1 -c ${CONDA_PREFIX})" == 1 ]]; then
+    GCC_INCLUDE_PATH=$(dirname $(find ${CONDA_PREFIX} -name cxxabi.h))
+    GCC_ARCH_INCLUDE_PATH=$(dirname $(dirname $(find ${GCC_INCLUDE_PATH} -name c++config.h)))
```

Comment:
How will this work when I use Conda, but not compilers from it?

Reply:
I don't like the dirname, find and searching for `cxxabi.h`. It seems very fragile. Ideally we would just use cmake, and have only minimal bash script to show how to use it.

Can this logic be moved inside cmake with some more clean approach?

Reply:
Sounds fair enough. I am not an expert on cmake, but I will give it a shot. Thanks for the review